SharedContent/cpp-winui-template/WinUI3TemplateCpp/SampleConfiguration.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+#include "pch.h"
+
+namespace winrt::WinUI3TemplateCpp
+{
+	class Settings
+	{
+	public:
+		static hstring FeatureName;
```

Comment:
Indentation consistency - use spaces, not tabs here, and only 4 spaces after a `{`